index.bs

```diff
@@ -284,31 +284,55 @@ mechanism for metadata discovery.
 Transport and metadata discovery with QUIC {#transport}
 =======================================================
 
-If a listening agent wants to connect to an advertising agent, or to
-learn further metadata about it, it initiates a [[QUIC]] connection to
-the IP and port from the SRV record.  Prior to authentication, a message may be
-exchanged (such as further metadata), but such info should be treated as
-unverified (such as indicating to a user that a display name of an
+If a listening agent ("controller") wants to connect to an advertising agent
```

Comment:
I'm sure Peter and/or Mark will chime in, but I think they have avoided controller/receiver terminology is because it seems Presentation-API-centric.  I'm neutral about this though.

Reply:
I'll leave as is for now, and bring it up with them.

Reply:
I like controller/receiver as terms in general, but in the context of mDNS and discovery, we can't assume the receiver is always the advertiser and the controller is always the listener.

Reply:
%s/controller/listener/g
%s/receiver/advertiser/g